build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -22,7 +22,11 @@
     </GetCurrentRuntimeInformation>
 
     <!-- Common Properties -->
-    <PropertyGroup>      
+    <PropertyGroup>
+      <IsDesktopAvailable>False</IsDesktopAvailable>
+      <IsDesktopAvailable Condition=" '$(OSName)' == 'win' ">True</IsDesktopAvailable>
+      <ExeExtension>.exe</ExeExtension>
```

Comment:
This is already in build.proj:

[https://github.com/dotnet/cli/blob/feature/msbuild/build.proj#L32-L33](https://github.com/dotnet/cli/blob/feature/msbuild/build.proj#L32-L33)